src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -54,7 +59,8 @@
  (fn [{:keys [db now]} [amount]]
    (let [wallet-address          (get-in db [:wallet :current-viewing-account-address])
          token                   (get-in db [:wallet :ui :send :token])
-         to-address              (get-in db [:wallet :ui :send :to-address])
+         account-address         (get-in db [:wallet :ui :send :send-account-address])
+         to-address              (or account-address (get-in db [:wallet :ui :send :to-address]))
```

Comment:
We can use the `not-found` in `get-in`

`(get-in db [:a :b :c] :return-if-not-found)`